### PR TITLE
added first draft of intro to schema and stmt on versioning

### PR DESCRIPTION
**Pull request by jeffreycwitt**
Added first ROUGH draft of "Intro to Schema" and "Stmt on Versioning". 

Please take a look when you get a chance and let me know what you think. 

If we like the draft, we can merge it to develop and make further revisions on new branches.

best,
jw

### CODE REVIEW

**Comment by nivaca** [comment]
Thanks. Looks very nice.

Nicolas Vaughan
Profesor Asistente
Departamento de Filosofía
Universidad de los Andes
Bogotá, Colombia

**Comment by nivaca** [comment]
PD. Why did you create a new branch for this?

Nicolas Vaughan
Profesor Asistente
Departamento de Filosofía
Universidad de los Andes
Bogotá, Colombia

File: docs/statement-on-versioning.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+The LombardPress-Web application makes different sets of stylesheets for each XML schema that the platform supports. These stylesheets transform the source edition to a common HTML format used for display within the framework and manipulation by the framework's javascript code. This allows the framework to support multiple XML schemas while at the same time allowing javascript code to be written in expectation of a single uniform DOM.
+
+A non-breaking change to the LombardPress-Schema (that is a move from version 1.0.0 to 1.1.0) would be a change to the schema that does not require the LombardPress-Web application to create a new set of style sheets to handle the new schema or to code conditionals within the existing style sheet. Rather, supporting such changes would only require (typically) the addition of new templates within existing XSLT stylesheet to handle the new features introduced in the 1.1.0 schema. A test of this compatibility is that this same stylesheet--now updated to handle 1.1.0 features--should be able to operate seamlessly on source files that remain at the 1.0.0 level.
```
Comment (stenskjaer) [inline docs/statement-on-versioning.md]: It is not entirely clear to me which type of changes in the schema this example generalizes to. So for instance, the changes discussed in #87, would they constitute a major change? My feeling is that it is, as such a change would modify the meaning of the `bibl` and `note` elements (or something similar). But is that intuition right?

File: docs/intro-to-lbp-schema.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+The LombardPress-Schema is one such known standard. The LombardPress-Schema pays close attention to the ontology, resources ids, and general recommendations of the SCTA and offers explicit guidelines about how these resources ids should be employed in the editing process. But the LombardPress-Schema need not be the only standard that the SCTA build script can accommodate. It is quite possible that someday a subdomain of scholastic texts will demand a schema that the LombardPress-Schema cannot accommodate. This new schema can make its own provisions for how to use the SCTA ontology to connect to the wider corpus. The SCTA build script can be then be modified to crawl two distinct, but standardized and documented, data serializations. Nevertheless, the LombardPress-Schema is, at the present, the most mature attempt both to provide a detailed encoding standard for all scholastic texts and to offer guidelines about how to prepare a text in such a way that it can be easily absorbed within the *Scholastic Commentaries and Texts Archive*.
+
+# Status and Future Development
```
Comment (stenskjaer) [inline docs/intro-to-lbp-schema.md]: This is a great section. I really like making the best practice principles explicit. But I'm in doubt if the heading reflects the content. Based on the headline I would expect a development roadmap or something similar. Should the heading be changed? "Development principles and best practices" or something?

File: docs/intro-to-lbp-schema.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+However, such cases of 'low hanging fruit' only constitute the first and easiest level of decision making when it comes to semantic markup of a complex text. The TEI consortium is the first to acknowledge this. It is precisely for this reason that the TEI guidelines include an astounding number (when compared to other XML schemas) of available elements. The TEI guidelines offers such a vast array of elements precisely because they expect further schema customization and documentation by smaller groups that aim to provide field standards for particular genres of texts and problem domains. Such customizations happen within the TEI consortium (for example, TEI-Simple, TEILite) and without (for example [EpiDoc](http://www.stoa.org/epidoc/gl/latest/)).
+
+The LombardPress-Schema is an attempt to provide one such customized schema for a particular problem domain. The primary focus of the LombardPress-Schema is to provide a uniform encoding standard for the editing of the medieval scholastic corpus (though its adoption is not limited to this domain), with a special emphasis on supporting critical and (semi) diplomatic transcriptions.
```
Comment (stenskjaer) [inline docs/intro-to-lbp-schema.md]: I took the liberty of separating this long paragraph. Undo if it's a bad idea.

**Pull request merged**